pkg/policy/api/rule.go

```diff
@@ -274,6 +274,9 @@ type EgressRule struct {
 // Example: 192.0.2.1/32
 type CIDR string
 
+// CIDRMatchAll is a []CIDR that matches everything
+var CIDRMatchAll = []CIDR{CIDR("0.0.0.0/0"), CIDR("::/0")}
```

Comment:
Can you change this name? It's not actually matching "All" it's matching all IPs that *don't* belong to the cluster CIDR, right?

Reply:
Yes but that's a datapath specific behaviour. The CIDR itself really matches everything. I know exactly what you mean but we should document the behaviour in one place instead of spreading it out. If we ever change it we have to touch every single place.

Reply:
That's true... But this needs to be documented what about in `pkg/policy/rule.go:362`? and `381`?

Reply:
Agreed